### ISSUE TITLE
Allow copying text from MusicBrainz ID Locator Failure Message

### ISSUE THREAD

**Issue by marknoble** [ZuneDev/ZuneModdingHelper#11]
When the MusicBrainz ID Locator generates a failure message, it is displayed in a scrollable panel, but the text from this cannot readily be copied elsewhere for analysis or documentation.

I'd suggest allowing this text to be copied.

**Comment by yoshiask** [comment]
I don't believe I can change whether that text is selectable. I can look into either adding a "copy text" button or a custom dialog I have more control over.

**Comment by yoshiask** [comment]
Selecting text in the dialog will now automatically copy it to your clipboard. I hope to improve this experience in the future with a button to open new GitHub issue with the error message already populated.
